charts/timescaledb-single/values.yaml

```diff
@@ -293,6 +290,10 @@ networkPolicy:
 # https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
 nodeSelector: {}
 
+# https://github.com/timescale/timescaledb-tune
+timescaledbTune:
+  enabled: False
```

Comment:
one optional flag we allow specified by `tune` is `max-connections`, so maybe we can expose that?

Reply:
I'm not opposed to setting max-connections, but I think it best if we have all the
explicit settings in the same spot, currently that's in :

https://github.com/timescale/timescaledb-kubernetes/blob/master/charts/timescaledb-single/values.yaml#L128

So I'm thinking of using timescaledb-tune only to autotune, not to set things explicitly, there's good ways of doing that already.
Plus, max-connections is almost never related to CPU/Memory limits/requests, mostly an application level thing.

Reply:
Happy to have some more flags here to control the behaviour